environments/stg/stg.tfvars

```diff
@@ -1664,5 +1664,20 @@ frontends = [
     custom_domain  = "em-icp.aat.platform.hmcts.net"
     backend_domain = ["firewall-prod-int-palo-cftaat.uksouth.cloudapp.azure.com"]
 
+  },
+  {
+    product          = "dss-update-case"
+    name             = "dss-update-case"
+    custom_domain    = "dss-update-case.aat.platform.hmcts.net"
+    backend_domain   = ["firewall-nonprodi-palo-cftaatappgateway.uksouth.cloudapp.azure.com"]
```

Comment:
I think this should be `"firewall-prod-int-palo-cftaat.uksouth.cloudapp.azure.com"`

Reply:
thanks @reespozzi , I have done the changes